tardis/tests/test_atomic.py

```diff
@@ -1,32 +1,122 @@
-from tardis import atomic
-from numpy import testing
-import pytest
 import os
+import pytest
+from numpy import testing
+from tardis import atomic
+
+
+chianti_he_db_h5_path = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)), 'data', 'chianti_he_db.h5')
+
 
-def test_atomic_h5_readin():
+def test_data_path():
+    data_path = atomic.data_path('test')
+    assert data_path.split('/')[-3:] == ['tardis', 'data', 'test']
+
+
+def test_read_basic_atom_data():
     data = atomic.read_basic_atom_data(atomic.default_atom_h5_path)
     assert data['atomic_number'][13] == 14
     assert data['symbol'][13] == "Si"
-    si_mass = data['mass'][13]
-    testing.assert_almost_equal(si_mass, 28.085, decimal=4)
-    pass
+    testing.assert_almost_equal(data['mass'][13], 28.085, decimal=4)
 
-def test_ionization_h5_readin():
+
+def test_read_ionization_data():
     data = atomic.read_ionization_data(atomic.default_atom_h5_path)
```

Comment:
I'd suggest using fixtures for the data paths.
That way they could easily be changed for all tests in case something changed.
This goes in line with the DRY principle.